web/lib/js/main.js

```diff
@@ -277,7 +284,8 @@ function refresh(data) {
         $(".distance").html(parseFloat(lowest_distance).toFixed(2));
 
         var paused = document.getElementById("player").paused;
-        if (!paused) {
+        if (!paused || !g_initial_received) {
```

Comment:
The player is paused initially, so need to perform at least one refresh at the start (that was my initial problem with the `master` version which I've mentioned earlier.